app/Models/Forum/Topic.php

```diff
@@ -80,8 +80,8 @@ public static function createNew($forum, $params, $pollParams = null)
             }
 
             if (($params['cover'] ?? null) !== null) {
```

Comment:
I think I did that somewhere else but the fact I forgot it already means I shouldn't use it.